index.js

```diff
@@ -34,6 +49,20 @@
 
 
 // ***** Question 4 *****
+
+function sameSameButDifferent(num, maybeNum){
+  if (num === maybeNum) {
+    return "same same"
+  }
+  else if (num == maybeNum){
+    return "same same (but different)"
+  }
+  else {
+    return "different"
+  }
+
+}
```

Comment:
Good! Another approach would be to convert the string to a number with parseInt to avoid `==` :

`} else if (num === parseInt(maybeNum)) {`